drivers/dma/dma_mcux_lpc.c

```diff
@@ -238,12 +243,23 @@ static int dma_mcux_lpc_queue_descriptors(struct channel_data *data,
 		/* Leave curr pointer unchanged so we start queuing new data from
 		 * this descriptor
 		 */
-		/* Enable interrupt and reload for the descriptor */
-		xfer_config = DMA_CHANNEL_XFER(1UL, 0UL, 1U, 0U,
-					width,
-					src_inc,
-					dest_inc,
-					MIN(local_block.block_size, max_xfer_bytes));
+		if (callback_en) {
+			/* Enable interrupt and reload for the descriptor */
+			xfer_config = DMA_CHANNEL_XFER(1UL, 0UL, 1U, 0U,
+						width,
+						src_inc,
+						dest_inc,
+						MIN(local_block.block_size, max_xfer_bytes));
+		} else {
+			/* Disable interrupt, we don't want an interrupt after this block
+			 * enable reload for the descriptor
+			 */
+			xfer_config = DMA_CHANNEL_XFER(1UL, 0UL, 0U, 0U,
```

Comment:
similar to other comment, can it just be `DMA_CHANNEL_XFER(1, 0, callback_en, ...)`

Reply:
Thanks, makes sense. I have made the change.